client/src/app/shared/shared-main/angular/number-formatter.pipe.ts

```diff
@@ -21,14 +16,25 @@ export class NumberFormatterPipe implements PipeTransform {
     return +f
   }
 
+  private dictionary: Array<{max: number, type: string}> = [
+    { max: 1000, type: '' },
+    { max: 1000000, type: 'K' },
+    { max: 1000000000, type: 'M' }
+  ]
+  constructor (@Inject(LOCALE_ID) private localeId: string) {}
+
   transform (value: number) {
     const format = this.dictionary.find(d => value < d.max) || this.dictionary[this.dictionary.length - 1]
     const calc = value / (format.max / 1000)
     const integralPart = Math.floor(calc)
     const decimalPart = NumberFormatterPipe.getDecimalForNumber(calc)
+    const decimalSeparator = Intl.NumberFormat(this.localeId)
```

Comment:
Shall I put this in a try/catch, so it'll be handled gently in unsupported browsers?